File: A/Shovel.java
import java.util.Scanner;

public class Shovel {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int price = scan.nextInt();
        int coins = scan.nextInt();
        int counter = 1;
        int temp = price;
        while (temp % 10 != 0 && temp % 10 != coins) {
            counter++;
            temp += price;
        }

        System.out.println(counter);
    }
}

File: CodeForces/Shovel.cpp
#include <iostream>
using namespace std;

int main(){
    int coins, curles;
    cin >> coins;
    cin >> curles;
    int counter = 1;
    int temp = coins;
    while(temp % 10 != 0 && temp % 10 != curles){
        counter++;
        temp = coins * counter;
    }
    cout  <<  counter << endl;
    return 0;
}